### DISCUSSION TITLE
Error Starting Kubernetes in mac monterey 12.5

### DISCUSSION THREAD

**Discussion by Darshan-tw** [rancher-sandbox/rancher-desktop#2638] [Q&A]
Getting error in mac after launching rancher desktop. Sharing the 2 screenshots for the reference 

I tried some suggested solutions but not working 

1. Tried "sudo mkdir -m 775 /private/var/run/rancher-desktop-lima" from  **https://github.com/rancher-sandbox/rancher-desktop/issues/1815**
2. Removing the ./kube/config
3. Tried factory reset many times. 
 
![Screenshot 2022-07-27 at 5 17 39 PM](https://user-images.githubusercontent.com/88665864/181241009-a2869074-a664-44dc-80ed-ae66d460af0f.png)
![Screenshot 2022-07-27 at 5 22 07 PM](https://user-images.githubusercontent.com/88665864/181241026-95ea90ed-3cb1-44ca-8679-d3ee65e80806.png)

**Comment by mook-as** [comment]
Hi there!
Would it be possible to attach your logs to help in troubleshooting?
- In the _Troubleshooting_ tab in the main window, check the _Enable debug mode_ checkbox.
- Reproduce the issue again.
- Locate the logs produced (there's a button near the previous checkbox to show it in Finder)
- Review the logs to ensure none of your credentials or secrets are in there before posting.

Thanks! Of particular interest is `lima.log`, though the other logs starting with `lima` may be helpful as well.
